validator/distro_validator.go

```diff
@@ -169,7 +170,7 @@ func validateMultipleProviderSettings(d *distro.Distro) ValidationErrors {
 		}
 		if err := settings.FromDistroSettings(*d, region); err != nil {
 			errs = append(errs, ValidationError{
-				Message: fmt.Sprintf("distro '%v' decode error: %v", distro.ProviderSettingsListKey, err),
+				Message: errors.Wrapf(err, "distro '%v' decode error", distro.ProviderSettingsListKey).Error(),
```

Comment:
We tend to avoid wrapping validation errors, since we're interested in the error itself, and the location it less important.